### PR TITLE
Hide early universities, colleges.

### PR DESCRIPTION
**Pull request by zerebubuth**
Connects to #1622.

### CODE REVIEW

File: yaml/pois.yaml [hunk 1]
```diff
@@ -562,14 +562,34 @@ filters:
...
+          - [ 11,  5000000 ]
+          - [ 12,  1000000 ]
+          - [ 13,   500000 ]
+          - [ 14,   500000 ]
+          - [ 15,   100000 ]
+        default: 16
     output:
       <<: *output_properties
-      kind: university
+      kind: { col: amenity }
```
Comment (nvkelso) [inline yaml/pois.yaml]: This is a no-op, and the old way is more legible.

File: yaml/pois.yaml [hunk 2]
```diff
@@ -562,14 +562,34 @@ filters:
...
+    min_zoom:
+      lookup:
+        key: { col: way_area }
+        op: '>='
+        table:
+          - [ 10, 10000000 ]
+          - [ 11,  5000000 ]
+          - [ 12,  1000000 ]
+          - [ 13,   500000 ]
+          - [ 14,   500000 ]
```
Comment (nvkelso) [inline yaml/pois.yaml]: Zoom 14 has same area threshold as zoom 13? Seems different than `tier3_min_zoom` breaks (below), which are also weird.

```
  - &tier3_min_zoom
    lookup:
      key: { col: way_area }
      op: '>='
      table:
        - [ 8,  10000000 ]
        - [ 9,   5000000 ]
        - [ 10,  1000000 ]
        - [ 11,   500000 ]
        - [ 12,   500000 ]
        - [ 13,   100000 ]
        - [ 14,    50000 ]
        - [ 15,     2000 ]
      default: 16
```

Comment (nvkelso) [reply yaml/pois.yaml]: Comparing tier3 and tier4, I think the following cuts are more consistent with old behavior?

```
- [ 10, 10000000 ]
- [ 11,  1000000 ]
- [ 12,   500000 ]
- [ 13,   100000 ]
- [ 14,    50000 ]
- [ 15,     2000 ]
```

File: yaml/pois.yaml [hunk 3]
```diff
@@ -562,14 +562,34 @@ filters:
...
       tier: 3
   # college
   - filter: {amenity: college}
-    min_zoom: { min: [ { max: [ { sum: [ { col: zoom }, 2.35 ] }, *tier3_min_zoom ] }, 16 ] }
+    min_zoom:
+      lookup:
+        key: { col: way_area }
+        op: '>='
+        table:
+          - [ 12,   600000 ]
```
Comment (nvkelso) [inline yaml/pois.yaml]: Similar comment to https://github.com/tilezen/vector-datasource/pull/1763/files#r244405058, though this one used a different _scale factor_ of **2.35** versus **2.55** so maybe that's why the values are less as proposed?

Comment (nvkelso) [reply yaml/pois.yaml]: I suspect we can just reuse:

```
- [ 12,   500000 ]
- [ 13,   100000 ]
- [ 14,    50000 ]
- [ 15,     2000 ]
```

File: yaml/pois.yaml (hunk 1 above)
Comment (zerebubuth) [reply yaml/pois.yaml]: Ooops, yeah. Thanks for spotting that. I had tried to merge the `university` and `college` parts, but it turned out to not be possible with the area thresholds we wanted and I must have forgotten to put that back the way it was.

File: yaml/pois.yaml (hunk 2 above)
Comment (zerebubuth) [reply yaml/pois.yaml]: Cool, thanks. Updated in 5af8d31.

Yup, the `tier3_min_zoom` was probably where I started from, and the repeated 500,000 was copy pasta from there. As written, it does mean that nothing in tier 3 will get a `min_zoom` of 12. Would we want to change that to 200,000 perhaps?

File: yaml/pois.yaml (hunk 3 above)
Comment (zerebubuth) [reply yaml/pois.yaml]: OK, thanks. Updated in 5af8d31.

File: yaml/pois.yaml (hunk 2 above)
Comment (nvkelso) [reply yaml/pois.yaml]: Yes, I'd be fine changing tier3 filter generically to `200,000` as you propose (as well).

**Comment by nvkelso** [comment]
Fix looks good, and I've filed https://github.com/tilezen/vector-datasource/issues/1769 for the general tier 3 fix.

**Pull request merged**